src/pages/index.astro

```diff
@@ -1,7 +1,7 @@
 <html lang="en" class="h-screen">
 <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/png" href="/favicon.ico" />
+    <link rel="icon" type="image/svg+xml" href="logo.svg" />
```

Comment:
needs leading slash I think to ensure it renders on pages other than home page

Reply:
@Padraic-O-Mhuiris

Reply:
Good catch